pages/example-3-missing-authorization-api-route/common.tsx

```diff
@@ -13,12 +13,21 @@ export default function Common(props: Props) {
 
       <ul>
         <li>
-          <LinkIfNotCurrent href="/example-3-missing-authorization-api-route/vulnerable">
-            Vulnerable
+          <LinkIfNotCurrent href="/example-3-missing-authorization-api-route/vulnerable-1">
+            Vulnerable-1
           </LinkIfNotCurrent>{' '}
           - API code:{' '}
           <code>
-            pages/api/example-3-missing-authorization-api-route/vulnerable.ts
+            pages/api/example-3-missing-authorization-api-route/vulnerable-1.ts
+          </code>
+        </li>
+        <li>
+          <LinkIfNotCurrent href="/example-3-missing-authorization-api-route/vulnerable-2">
+            Vulnerable-2
+          </LinkIfNotCurrent>{' '}
+          - API code:{' '}
+          <code>
+            pages/api/example-3-missing-authorization-api-route/vulnerable-2.ts
```

Comment:
I guess this file doesn't exist, right? (Same with the API code link above)

Reply:
sure, my bad. copy-pasted too fast ;-)